sources/web/datalab/polymer/modules/template-manager/template-manager.ts

```diff
@@ -92,15 +92,43 @@ class TemplateManager {
 
   public static async newNotebookFromTemplate(template: NotebookTemplate) {
 
+    const appSettings = await SettingsManager.getAppSettingsAsync();
+
+    // TODO(jimmc): Look for a user preference for tempRoot
+    let tempRoot;
+    if (appSettings.defaultFileManager) {
```

Comment:
These 7 lines can be shortened: `let tempRoot = appSettings.DefaultFileManager || 'drive' + ':'`.

Reply:
Done.